test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
We don't use this test folder in our project, so add this folder to .gitignore and run command.

Reply:
done